pages/_app.js

```diff
@@ -0,0 +1,39 @@
+import '@/styles/globals.css'
+import Script from 'next/script';
+import Head from 'next/head';
+import { Fragment } from 'react'
+import { Poppins } from "@next/font/google";
+
+const poppins = Poppins({
+    weight: ["400", "700"],
+    style: ["normal", "italic"],
+    subsets: ["latin"],
+});
+
+
+export default function App({ Component, pageProps }) {
+  return (
+      <Fragment>
+          <div className={poppins.className}>
+              <Head>
+                  <title>Quizard</title>
+                  <meta
+                      name="Students aids"
```

Comment:
This sounds off... please explain why we need it, and why it's called "Students aids"

Reply:
These are just for the start they can and will be changed later when the SEO are ready.